codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/impl/SourceGenerator.scala

```diff
@@ -49,7 +49,26 @@ object SourceGenerator {
   /**
    * Generate the 'managed' code for this model: code that will be regenerated regularly in the 'compile' configuratio
    */
-  def generateManagedTest(model: ModelBuilder.Model): Seq[File] = Seq.empty
+  def generateManagedTest(model: ModelBuilder.Model): Seq[File] = {
+    model.services.values
+      .flatMap {
+        case service: ModelBuilder.EntityService =>
+          model.lookupEntity(service) match {
+            case entity: ModelBuilder.ValueEntity =>
+              ValueEntityTestKitGenerator.generateManaged(entity, service)
```

Comment:
but perhaps name it `generateManagedTest` to make it clear this generates files for the 'test' scope?